response_operations_ui/templates/account/verify-account.html

```diff
@@ -0,0 +1,132 @@
+                {% set header_text = 'This page has ' ~ error_count ~ ' error' ~ plural_suffix %}
+            {% call
+                onsPanel({
+                    "title": header_text,
+                    "type": "error"
+                })
+            %}
+                <p>These <strong>must be corrected</strong> to continue.</p>
+                {% for error in form.password.errors %}
+                    {% do errorData.append(
+                        {
+                            "text": error,
+                            "index": true,
+                            "url": "#password",
+                            "classes": "ons-js-inpagelink"
+                        }
+                    ) %}
+                {% endfor %}
+                {% for error in form.password_confirm.errors %}
+                    {% do errorData.append(
+                        {
+                            "text": error,
+                            "index": true,
+                            "url": "#password_confirm",
+                            "classes": "ons-js-inpagelink"
+                        }
+                    ) %}
+                {% endfor %}
+                {{
+                    onsList({
+                        "classes": "list--bare",
+                        "itemsList": errorData
+                    })
+                }}
+            {% endcall %}
+        {% endif %}
+            <form method="post" action="" class="form" role="form">
+                <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+                <h1>Verify account</h1>
+                <p>An account has been created for {{ username }}. To activate the account, you must create a password.</p>
+                <span><strong>Username</strong></span>
+                <p>{{ username }}</p>
+                {% from "components/panel/_macro.njk" import onsPanel %}
+                {{
+                    onsPanel({
+                        "spacious": true,
+                        "body": '<p>Your password must have:</p><ul>
+                        <li>at least 12 characters</li>
+                        <li>at least 1 uppercase letter</li>
+                        <li>at least 1 symbol (eg: ?!£%)</li>
+                        <li>at least 1 number</li></ul>'
+                    })
+                }}
+                {{
+                    onsPassword({
+                        "id": "password",
+                        "name": "password",
+                        "label": {
+                            "text": 'New password'
+                        },
+                        "attributes": {
+                            "autocomplete": 'off'
+                        },
+                        "showPasswordText": "Show password",
+                        "error": passwordError
+                    })
+                }}
+                {{
+                    onsInput({
```

Comment:
I am thinking this should also have show password?

Reply:
Agreed, fixed.